packages/exchange/src/exchange/providers/groq.py

```diff
@@ -0,0 +1,97 @@
+import os
+from typing import Any, Dict, List, Tuple, Type
```

Comment:
:nit: i think our patterns here don't realize as of python 3.9+ we generally don't need `Dict`, `List`, and `Tuple` anymore we can use the primitives (`dict`, `list`, and `tuple` respectively)

Reply:
https://github.com/block-open-source/goose/pull/149 to do this everywhere